autoextract_spiders/spiders/autoextract_spider.py

```diff
@@ -32,6 +34,11 @@ def __init__(self, url, **kwargs):
             if page_type:
                 self.meta['autoextract']['pageType'] = page_type
 
+        # Request fingerprint for Frontera requests
+        if meta.get('cf_store') and 'source_url' in meta:
+            fp = request_fingerprint(self).encode()
+            meta['frontier_fingerprint'] = hashlib.sha1(fp).hexdigest()
```

Comment:
@croqaz I saw this approach in a few projects, wonder is it really needed? I mean, afaik Frontera [takes](https://github.com/scrapinghub/scrapy-frontera/blame/master/README.rst#L14) Scrapy request fingerprint, and by default it's already sha1. Do we need to redefine it here?

Reply:
Yes, this is not necessary because the request is already fingerprinted 👍